packages/story-editor/src/components/floatingMenu/elements/flipHorizontal.js

```diff
@@ -24,13 +24,16 @@ import { __ } from '@googleforcreators/i18n';
  * Internal dependencies
  */
 import { IconButton } from './shared';
+import useFlip from './shared/useFlip';
 
 function FlipHorizontal() {
```

Comment:
Based on the grooming discussion, updated this existing issue: https://github.com/GoogleForCreators/web-stories-wp/issues/6690

We can define it better once the menu bar is in place.

Reply:
^ cc @littlemilkstudio for visibility since I know you also had some ideas for improving the Design Panel performance specifically, or were planning to work on it. We were discussing today in the grooming to follow up later (#6690) with using similar structure/logic for the design panel as Morten is doing here in this PR (reusing the same components as well) for not duplicating the buttons + improving performance.

Reply:
@miina thanks for tagging. Yea, I haven't started looking into performance wins for the design panel yet, was kinda waiting for the dust to settle here before I started. Happy to audit any PRs for performance improvements if you'd like.

The only goal there is to not have the entire panel re-render every time any input updates. I'm pretty fine with whatever architectural decisions get us there. I know the logic with all the pushUpdate stuff is pretty complex so it might make it tough.